### PR TITLE
Remove fallback resource

### PR DESCRIPTION
**Pull request by elinor-fung**
The 'small' fallback resource on Windows is a remnant of Silverlight and no longer needed.

Remove the fallback and rename the default to be `mscorrc` instead of `mscorrc.debug`.

### CODE REVIEW

**Review by jkotas — APPROVED** [review]
Thank you!

File: src/coreclr/tests/src/performance/Scenario/JitBench/Utilities/DotNetSetup.cs [hunk 1]
```diff
@@ -201,7 +201,7 @@ public static string GetNormalizedOSName(OSPlatform os)
             }
         }
 
-        
+
```
Comment (janvorli) [inline src/coreclr/tests/src/performance/Scenario/JitBench/Utilities/DotNetSetup.cs]: A nit - an unnecessary extra blank line

**Review by janvorli — APPROVED** [review]
LGTM, thank you!

**Comment by jkoritzinsky** [comment]
I believe this fixes https://github.com/dotnet/runtime/issues/12334

**Pull request merged**